src/GamemasterInGame/ShareButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button } from "@radix-ui/themes";
+
+interface ShareProps {
+  label: string;
+  url: string;
+  text: string;
+  title: string;
+}
+
+function ShareButton({ label, url, text, title }: ShareProps) {
+  const handleShare = async () => {
+    if (navigator.share) {
+      try {
+        await navigator.share({ url, title, text });
+      } catch (error) {
+        console.log("Failed to share.");
+      }
+    } else {
+      navigator.clipboard.writeText(url);
```

Comment:
Need to check that can write to clipboard, but permissions checking is blocked by me.

Reply:
Should also send toast on copy to clipboard.